src/library_openal.js

```diff
@@ -1914,6 +1916,12 @@ var LibraryOpenAL = {
     if (c.splitterNode) c.splitterNode.disconnect();
     // May happen if user hasn't decided to grant or deny input
     if (c.scriptProcessorNode) c.scriptProcessorNode.disconnect();
+    if (c.mediaStream) {
+      // Disabling the microphone of the browser.
```

Comment:
please expand this comment, specifically why do we need to disable the microphone manually this way. I am unclear on that myself, so I don't have a suggestion for better text here.

Reply:
done.